module-5.md

```diff
@@ -12,36 +12,102 @@ Planned:      1.25 hour
 
 ## 1: Topic introduction
 
-
+* Take up homework 
+* Discuss the need for keeping a pulse of the network
+  * What attributes are important to monitor
+  * How are these attributes measured?
+  * Identify units and what they mean
+  * Where have we seen it in our daily lives?
 
 ## 2: Hands-on and material exploration
 
-Monitor flow of network traffic using different tools as we do everyday digital things.
+Monitor flow of network traffic using different tools as we do every day digital things.
 
 ### Objectives
 
+Learn how these metrics affect us in our day to day lives, and in contrast how the same metrics affect others.
 
 ### Materials
 
+* Your cellphone
+* Some sort of public WiFi or some sort of internet connection
+* A Raspberry Pi 3 for each member labeled with the SSID for the node
+* SD cards with pre-flashed images with all required software
+* Laptops that can establish a SSH session to each Raspberry Pi (each running a Host AP with unique SSID)
+* 802.11s capable (e.g. TP-Link TL-WN722N v1) Radio for each Raspberry Pi 3
 
 ### Format
 
+Groups (or pairs) to start, class discussion then back to groups again
 
 ### Activity
 
-- Understanding the current internet:
-  - Speed test with library internet
-  - How much data does your app use
-
-- Grafana to show bandwidth usage:
-  - Locally stream music and video and see interface bandwidth
-  - What do your metrics mean?
-
-- Physical network topology (i.e. Toronto Mesh map) and Virtual peering topology (i.e. fc00.org)
+* Review your cell phone usage
+  * Identify your apps’ data usage
+  * Record top 3 apps that took up the most data
+  * Was this data cellular, or WiFi?
+* Public WiFi Speed Test
+  * Use a speed test tool to test the speed of the public WiFi, record the result.
+  * Is it faster or slower than the one you have at home? 
+  * Run the speed test again, record the result
+  
+* Discuss
+  * What unit is the data usage of your apps measured in?
+  * What does MB mean?
+  * What was your top app and how much MB did it use?
```

Comment:
This should be voluntarily reported as it's like asking someone's browsing history.

Reply:
I don't think its like some ones browser as its not detail information. Just general "Firefox used 20 megs" not *insert incriminating items here*
Bur regardless if some one does not  feel comfortable showing their top app, or any apps for that matter they dont have to.  

The point is to open the eyes to the general public of how much information these applications actually exchange.  Most people never look at this and would be shocked to see actually how much data an app like Instagram actually takes up!